terraform/provider.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  azure_credentials      = try(jsondecode(var.azure_sp_credentials_json), null)
+  infrastructure_secrets = yamldecode(data.azurerm_key_vault_secret.secrets["INFRASTRUCTURE"].value)
```

Comment:
INFRASTRUCTURE secret currently contains find a lost trn app env variables. We need another YAML secret for QTS.

Reply:
@merrilmathew :+1:

By the way, to generate a secret, you can run the `rails secret` command from the project root (you need to have ruby/rails set up locally)